lochness/mediaflux/__init__.py

```diff
@@ -148,6 +153,9 @@ def sync_module(Lochness: 'lochness.config',
                                                 BIDS=Lochness['BIDS']) / \
                                                         subpath.parent)
 
+                            mf_local = str(Path(mf_local) / prod['out_dir']) \
+                                    if 'out_dir' in prod else mf_local
```

Comment:
This check can be insufficient. If someone writes `out_dir` but does not assign a value to it, the `if` will still evaluate to true. So we should need:

```python
> if ('out_dir' in prod and prod['outdir']) else mf_local
```

Reply:
While I wait for your reply, I propose this concise code:

```python
mf_local = tree.get(datatype,
              subj_dir,
              processed=processed,
              BIDS=Lochness['BIDS']) / subpath.parent

if 'out_dir' in prod and prod['out_dir']:
    mf_local = mf_local / prod['out_dir']

mf_local= str(mf_local)
```

Reply:
is 'outdir' typo there?

Reply:
Okay I have made them all `out_dir`. Sorry.

Reply:
Empty `out_dir` in the configuration file should appear quite obvious to the user. If it's `out_dir` is empty indeed, it will not have any effect on the output `mf_local` variable. 

```
                interviews:
                    - product: open
                      data_dir: PrescientXX_Interviews/OPEN
                      out_dir: 
                      pattern: '*'
```